### ISSUE TITLE
Why use train and validation datasets both for training?

### ISSUE THREAD

**Issue by Songyeyaosong** [csuhan/s2anet#147]
Normally we train with a training set, then tune parameters with a validation set, and finally test generalization performance with a test set? So... I donot understand why.
